pkg/html/releasehtml/html.go

```diff
@@ -40,6 +40,12 @@ const (
 #table td, #table th {
 	border: 
 }
+
+.collapsing {
+    -webkit-transition: none;
+    transition: none;
+    display: none;
+}
```

Comment:
if we do it, i'd just put it in the generichtml HTMLPageStart since i think then any page that does this would be covered.